typescript/web/src/components/workspaces/workspaces.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button, Center, chakra, Flex, Heading, Text } from "@chakra-ui/react";
+import { isEmpty } from "lodash";
+import { FC, useCallback, FormEvent } from "react";
+import NoWorkspacesGraphics from "../graphics/no-workspace";
+import {
+  useWorkspaceNameInput,
+  WorkspaceNameInput,
+  WorkspaceNameInputProvider,
+  WorkspaceNameMessage,
+} from "../workspace-name-input";
+import { useCreateWorkspace } from "../workspace-switcher/create-workspace-modal";
+import {
+  WorkspacesContextProvider,
+  WorkspacesProps,
+} from "./workspaces-context";
+import { WorkspacesList } from "./workspaces-list";
+
+const ChakraNoWorkspaces = chakra(NoWorkspacesGraphics);
+
+const CreateButton: FC<{ isDisabled?: boolean }> = ({ isDisabled }) => (
+  <Button
+    width="full"
+    type="submit"
+    isDisabled={isDisabled}
+    colorScheme="brand"
+    aria-label="create workspace button"
+    mt="8"
```

Comment:
What's this margin useful for? Shouldn't it be handled at an upper-level with a `VStack` and some spacing?

Reply:
I just want to push down the button, not space every element in the form